src/main/main.go

```diff
@@ -33,15 +34,25 @@ func main() {
 	}
 
 	krakenAPI := krakenapi.New(args.Key, args.Secret)
-	var notifierInstance *notifier.Notifier
-	if args.CredentialsFile != "" {
-		var gmailer notifier.Notifier = notifier.MustNewGMailer(args.CredentialsFile, "me")
-		notifierInstance = &gmailer
+	var notifiersInstance []*notifier.Notifier
+
+	if args.EmailCredentialsFile != "" {
```

Comment:
Could be nice to have some factory pattern for this? (in case we add more notifiers)

Reply:
I am not sure if I understand. I am new to `golang`. I thought the following logic is a factory pattern that creates concrete notifiers that conform to the `Notifier` interface.

Reply:
Actually yeah it already is pretty much, was just thinking it could be a separate class. But it's not a big deal